etcd_helper/etcdfuncs.go

```diff
@@ -0,0 +1,113 @@
+package etcd_helper
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/coreos/etcd/client"
+)
+
+type Etcdreader struct {
+	etcdServiceURL string
+}
+type Etcdwriter struct {
+	etcdServiceURL string
+}
+
+func (o Etcdreader) get(resourceKey string) (string, error) {
+	cfg := client.Config{
+		Endpoints: []string{o.etcdServiceURL},
+		Transport: client.DefaultTransport,
+	}
+	c, err := client.New(cfg)
+	if err != nil {
+		log.Fatal(err)
+	}
+	kapi := client.NewKeysAPI(c)
+
+	resp, err1 := kapi.Get(context.Background(), resourceKey, nil)
+	if err1 != nil {
+		return "", fmt.Errorf("Error: %v", err1)
+	} else {
+		//log.Printf("%q key has %q value\n", resp.Node.Key, resp.Node.Value)
+	}
+	return resp.Node.Value, nil
+}
+
+func (o Etcdreader) getList(resourceKey string) ([]string, error) {
+	cfg := client.Config{
+		Endpoints: []string{o.etcdServiceURL},
+		Transport: client.DefaultTransport,
+	}
+	c, err := client.New(cfg)
+	if err != nil {
+		return nil, fmt.Errorf("Error: %v", err)
+	}
+	kapi := client.NewKeysAPI(c)
+
+	var currentListString string
+	var operatorList []string
```

Comment:
We should not have any variable name that use the word 'operator'. That way the library will keep its generic nature. Also, the word 'operator' has no significance in this layer.

Reply:
This is still remaining. You can change operatorList to currentList.